repos/aspnetcore.proj

```diff
@@ -1,4 +1,7 @@
 <Project>
+  <PropertyGroup>
+    <SourceDirectory>aspnetcore</SourceDirectory>
```

Comment:
Not necessary when the same as the project name. Not worth respinning to remove it though, IMO. Will be nice when we can fix up cloning. (https://github.com/dotnet/source-build/issues/1534 / #1571)